### PR TITLE
PR for restructuring of repository

### PR DESCRIPTION
**Pull request by auringonnousu**
Please check before you submit a PR:

- [x] Have you checked to ensure there aren't other open PR for the same update/change?
- [x] Have you read the CONTRIBUTING document?
- [x] Does the commit message follow our guidelines?
- [ ] Have you done your changes in a separate branch?
- [x] Give a descriptive title to you PR.
- [x] Give a description of your changes.

Restructuring the repository by adding new folders and reorganizing the existing models into:

supervised learning > classification > knn
supervised learning > regression > linear model
New folder structure:

Folder supervised learning with subfolders for classification, regression and images_text
Folder unsupervised learning
Folder reinforcement learning
Update of README.md accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,74 @@
...
+---
+Please read the [Contributing Guidelines](#contributing) if you want to contribute.
+
+Feel free to open a GitHub issue if you have any questions. 
+
 
 ## License
-Teaching material is made availably under a [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/deed.en) license.  
+---
+Please read the [License](/Machine-Learning-OER-Basics-main/License.md) for more information.
```
Comment (benjwolff) [inline README.md:66]: The file "License.md" does not exist in the repo.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,74 @@
...
+pip install -r requirements.txt
+```
+
+Or you can try out [Binder](https://mybinder.readthedocs.io/en/latest/) by clicking on the following badge:
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Machine-Learning-OER-Collection/Machine-Learning-OER-Basics/HEAD)
+
+## Contributing 
+---
+Please read the [Contributing Guidelines](#contributing) if you want to contribute.
```
Comment (benjwolff) [inline README.md]: This link should be fixed.

**Review by benjwolff — COMMENTED** [review]
@auringonnousu Please see my comments in the code

File: README.md (hunk 1 above)
Comment (benjwolff) [reply README.md:66]: @auringonnousu Now we have LICENSE and License.md. Anyways we should go over the licensing part at the very end.

Comment (auringonnousu) [reply README.md:66]: Will be fixed with adding .txt for each license in folder LICENSES

Comment (benjwolff) [reply README.md:66]: Maybe than better stick with LICENSE and change the according Link to LICENSE as well instead of introducing a new file?

Comment (benjwolff) [reply README.md:66]: Ok, got it. LICENSE is for the Licensetext(CC-BY 4.0) and License.md for an complete overview.

**Pull request merged**